www/templates/default/html/js/manager.js

```diff
@@ -1,3 +1,7 @@
+require([frontend_url + 'templates/default/html/js/vendor/select2/js/select2.min.js', 'jquery'], function(select2) {
+	WDN.jQuery(".use-select2").select2();
```

Comment:
So close, but not quite. Also, if you don't need the actual select2 (module), you can remove that param from the loader callback. The key here is _never_ assume a global variable exists inside the loader callback.

```
require(['jquery', frontend_url + 'templates/default/html/js/vendor/select2/js/select2.min.js'], function($) {
// $ is now the jQuery I'm looking for!!
})
```